dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScopeTest.kt

```diff
@@ -108,12 +112,17 @@ internal class RumSessionScopeTest {
 
     lateinit var fakeInitialViewEvent: RumRawEvent
 
+    @Mock
+    lateinit var mockSessionReplayFeatureScope: FeatureScope
+
     @BeforeEach
     fun `set up`(forge: Forge) {
         fakeInitialViewEvent = forge.startViewEvent()
 
         whenever(mockParentScope.getRumContext()) doReturn fakeParentContext
         whenever(mockChildScope.handleEvent(any(), any())) doReturn mockChildScope
+        whenever(mockSdkCore.getFeature(RumSessionScope.SESSION_REPLAY_FEATURE_NAME))
+            .thenReturn(mockSessionReplayFeatureScope)
```

Comment:
minor suggestion, just for the consistency

```suggestion
            doReturn mockSessionReplayFeatureScope
```